geomesa-spark/geomesa-spark-accumulo/pom.xml

```diff
@@ -101,6 +105,11 @@
                                     <pattern>org.apache.commons.codec</pattern>
                                     <shadedPattern>org.locationtech.geomesa.shaded.org.apache.commons.codec</shadedPattern>
                                 </relocation>
+                                <!-- shade thrift because spark depends on 0.9.2 and accumulo 1.8 depends on 0.9.3 -->
+                                <relocation>
+                                    <pattern>org.apache.thrift</pattern>
+                                    <shadedPattern>org.lt.geomesa.shaded.o.a.thrift</shadedPattern>
```

Comment:
do we care about consistent shading packages? (above we don't use abbreviations)